pkg/controller/humiocluster/pods.go

```diff
@@ -74,7 +74,7 @@ func constructPod(hc *corev1alpha1.HumioCluster, humioNodeName string, dataVolum
 			InitContainers: []corev1.Container{
 				{
 					Name:  "zookeeper-prefix",
-					Image: "humio/humio-operator-helper:dev",
+					Image: "humio/humio-operator-helper:0.0.3",
```

Comment:
Why does this PR show the image version changing? Looking at master, this `0.0.3` tag is already the one we use. Is it just GitHub being weird?

Reply:
I think I screwed up when I squashed it. fixing.